scripts/get_hashsets.js

```diff
@@ -31,11 +33,27 @@ function getBreachHashset(breach) {
       url,
       headers,
     };
-    const BREACH_HASHSET_ZIP = path.join(BREACH_HASHSET_DIR, `${breach.Name}.zip`);
 
     console.log(`Active, verified breach with email addresses: ${breach.Name}`);
     console.log(`Fetching ${url}...`);
-    request(hashsetRequestObject).pipe(fs.createWriteStream(BREACH_HASHSET_ZIP));
+
+    request(hashsetRequestObject, (error, response, body) => {
+      if (response.statusCode === 200) {
+        console.log("Uploading to S3 ...");
+        const uploadParams = {
+          Bucket: BREACH_HASHSET_BUCKET_NAME,
+          Key: `${breach.Name}.zip`,
+          Body: body,
+        };
+        s3.upload(uploadParams, (err, data) => {
```

Comment:
Not sure this improves anything, but possibly interesting to note (and may be related to #30)... But it may be worth investigating AWS S3 Promise support. After a bit of Googling, I found https://aws.amazon.com/blogs/developer/support-for-promises-in-the-sdk/

But not sure if it improves anything really:

```js
s3.upload(uploadParams).promise()
  .then(data => console.log(`Uploaded to ${data.Location}`);
  .catch(err => console.error(`err: ${err}`);
```

Or if we're trying to get away from Promises and into `await..async`:

```js
try {
  const data = await s3.upload(uploadParams).promise();
  console.log(`Uploaded to ${data.Location}`);
} catch (err) {
  console.error(`err: ${err}`);
}
```

Reply:
Updated to use s3 promise.